source/Octopus.Tentacle.Tests.Integration/Support/Logging/TestContextConnectionLog.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using Halibut.Diagnostics;
+using Halibut.Diagnostics.LogWriters;
+using Halibut.Logging;
+using Octopus.Tentacle.Tests.Integration.Util;
+using Serilog;
+using ILog = Halibut.Diagnostics.ILog;
+
+namespace Octopus.Tentacle.Tests.Integration.Support.Logging
+{
+    public class TestContextConnectionLog : ILog, ILogWriter
```

Comment:
We use this for the legacy builder. But would there be value in using this for the regular builder?

This is here for the `logLevel`, but I think it's also used for logging some of the other things, like endpoint etc.

It might not be useful in `ClientAndTentacleBuilder`, but I thought it worth an ask 😁

Reply:
It definitely would be. All the plumbing is there, so it should be easy to add it to the ClientAndTentacleBuilder when someone needs it. I didn't just because I'm just trying to get green and diagnose the flakey tests we see now.